Makefile

```diff
@@ -132,6 +132,26 @@ gotest: failpoint-enable
 	$(GOTEST) -ldflags '$(TEST_LDFLAGS)' $(EXTRA_TEST_ARGS) -cover $(PACKAGES_TIDB_TESTS) -coverprofile=coverage.txt -check.p true > gotest.log || { $(FAILPOINT_DISABLE); cat 'gotest.log'; exit 1; }
 	@$(FAILPOINT_DISABLE)
 
+gotest_in_verify_ci_part_1: failpoint-enable tools/bin/gotestsum tools/bin/gocov tools/bin/gocov-xml
+	@echo "Running in native mode."
+	@mkdir -p $(TEST_COVERAGE_DIR)
+	@export log_level=info; export TZ='Asia/Shanghai'; \
+	CGO_ENABLED=1 tools/bin/gotestsum --junitfile "$(TEST_COVERAGE_DIR)/tidb-junit-report.xml" -- -v -p $(P) --race \
+	-ldflags '$(TEST_LDFLAGS)' $(EXTRA_TEST_ARGS) -coverprofile="$(TEST_COVERAGE_DIR)/tidb_cov.unit_test.out" \
+	$(PACKAGES_TIDB_TESTS_EXPENSIVE) -check.p true || { $(FAILPOINT_DISABLE); exit 1; }
```

Comment:
Could we put this logic to Jenkins like the unit test in jenkins：https://github.com/PingCAP-QE/ci/blob/main/jenkins/pipelines/ci/tidb/tidb_ghpr_unit_test.groovy#:~:text=grep%20%22%5C%24%7Bpackage_base%7D/sessionctx%22%20packages.list%20%3E%3E%20%20packages.list.short?

Reply:
I think it is hard to maintain in jenkins file and contributors do not know where to modify it.

Reply:
I think put his logic in makefile will be more reasonable, it save a lot time when we try to integration it to daily-ci or merge-ci.